exist-core/src/test/resources-filtered/conf.xml

```diff
@@ -888,6 +888,10 @@
             raise-error-on-failed-retrieval="no">
         
         <builtin-modules>
+
+            <!-- XQSuite Test Framework for XQuery testing -->
```

Comment:
Will this now effectively add XQSuite to the XQuery contexts of the next release?

Reply:
No.